<launch>

  <!-- Launch the parking controller on the real robot. Launches safety controller, controller node, cone detection node, and the homography transformer -->
  <!-- Edit this to launch your safety controller you implemented in Lab 3! -->
  <!--node pkg="safety_controller" name="safety_controller" type="safety_controller.py" output="screen">
	<rosparam command="load" file="/../../safety_controller/params.yaml"/>
  </node-->

<!--  <node pkg="final_challenge_city" name="Trajectory_follower" type="pure_pursuit.py" output="screen">-->
<!--  	<param name="drive_topic" value="/vesc/ackermann_cmd_mux/input/navigation" />-->
<!--    <param name="odom_topic" value="/pf/pose/odom" />-->
<!--  </node>-->

  <node pkg="final_challenge_city" type="pure_pursuit.py" name="Trajectory_follower" output="screen">
		<param name="odom_topic" value="/pf/pose/odom"/>
        <param name="drive_topic" value="/drive" />
	</node>

<!--  <node pkg="final_challenge_city" name="stop_detector" type="stop_detector.py" output="screen"/>-->

  <node pkg="final_challenge_city" name="homography_transformer" type="homography.py" output="screen"/>

  <node pkg="final_challenge_city" name="stopping_controller" type="stopping_controller.py" output="screen"/>

</launch>
